RafRate.API/Configuration/Extentions/DataBaseExtension.cs

```diff
@@ -0,0 +1,13 @@
+﻿using RafRate.Data.Contexts;
+using Microsoft.EntityFrameworkCore;
+
+namespace RafRate.API.Configuration.Extentions;
+
+public static class DataBaseExtension
+{
+    static ConnectionStringContext stringContext = new();
+    
+    public static IServiceCollection AddDataBase(this IServiceCollection services)
+    => services.AddDbContext<AppDbContext>(options => 
+        options.UseSqlServer(stringContext.ConnectionString));
```

Comment:
ConnectionStringContext will only have 1 method for the rest of its life, and this method will only have 1 usage for the rest of its life. THere is no need to extract it into a separate method in a separate class. Just use `GetEnvironmentVariable` here. If your connection string will be null you will find out anyways so no need for the exception as well.

Reply:
The string was moved to a separate class on the advice @bogomazdmitry 
FIGHT GAGAGAGA

Reply:
![image](https://user-images.githubusercontent.com/36763052/184547452-ea3cca80-cc46-4667-9110-a9cf47203b1d.png)